app/auth/authorize_requests.rb

```diff
@@ -0,0 +1,29 @@
+class AuthorizeRequests
+  def initialize(headers = {})
+    @headers = headers
+  end
+
+  def call
+    {
+      user: user
+    }
+  end
+
+  private
+
+  attr_reader :headers
```

Comment:
Should be at the top

Reply:
Working on the feedback

Reply:
Same thing here as above. We only need the `headers` in our private methods.